hist/histpainter/src/THistPainter.cxx

```diff
@@ -4569,6 +4569,7 @@ void THistPainter::Paint(Option_t *option)
       return;
    }
 
+   gPad->RangeAxis(); //emit RangeAxisChanged() signal to sync axes
```

Comment:
Why don't you call RangeAxisChanged directly ?

Reply:
This compilation error: 

`error: no member named 'RangeAxisChanged' in 'TVirtualPad'
   gPad->RangeAxisChanged();
   ~~~~  ^
1 error generated.
`

I guess because RangeAxisChanged() is only defined for a TPad not a TVirtualPad?

Reply:
In that case it might be simply to define `RangeAxisChanged` in `TVirtualPad` than creating a new method which does simply a call to RangeAxisChanged ?

Reply:
Ok sure, I have changed it to that now